### DISCUSSION TITLE
Css not reflecting on my html webpage

### DISCUSSION THREAD

**Discussion by bariya001** [community/community#22681] [Other Feature Feedback, Questions, & Ideas]
<p>hello, i am just getting myself around web development and i recently tried publishing a webpage using repository and after all the files upload, only my html files showed, my css file wasn’t implemented but in my local host everything seem to be fine when i load the index file. here is the link  <a href="https://bariya001.github.io/bariya/" rel="nofollow noopener">https://bariya001.github.io/bariya/</a></p>
<p>how do i go about fixing this please?</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/bariya001">@bariya001</a>, welcome to the GitHub Support Community!</p>
<p>The issue appears to be the capitalization of your CSS folder. In the HTML page, you’re referencing <code>css/styles.css</code>, however, the CSS folder is named Css in your repository, with a capital C. File and Folder names are case-sensitive, so renaming the folder or changing the <code>href</code> value for your <code>&lt;link&gt;</code> element so that they match each other’s case should resolve the problem.</p>
<p>You can also enable this same functionality on windows, so that you can catch things like this before hand. Please see <a class="mention" href="/tajmone">@tajmone</a>’s post for more info on that:</p><aside class="quote quote-modified" data-post="3" data-topic="127320">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/css-not-being-applied/127320/3">CSS Not being applied</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Pages—websites for you and your projects hosted right on your GitHub repository.">GitHub Pages</span></a>
  </div>
  <blockquote>
    Hi <a class="mention" href="/mandrake-ux">@Mandrake-ux</a> and welcome on board! 
<a class="mention" href="/sethclydesdale">@sethclydesdale</a> is right, GH servers being Linux based have a case-sensitive file system. 
Since you didn’t spot the problem locally, I’m assuming you’re working under Windows OS. 
With Windows 10 you can now enable case-sensitive behavior on  a per-directory basis, which is always a good solution when working with cross-platform projects on GitHub: 

<a href="https://www.howtogeek.com/354220/how-to-enable-case-sensitive-folders-on-windows-10/" rel="noopener nofollow ugc">How-To-Geek » How to Enable Case Sensitive File and Folder Names on Windows 10</a>

I recommend you apply case-…
  </blockquote>
</aside>

**Comment by bariya001** [comment]
<p><a class="mention" href="/SethClydesdale">@SethClydesdale</a> you were spot on with that. i corrected the typo and its all good with the site now. here is the corrected link  <a href="https://bariya001.github.io/personal/" rel="nofollow noopener">https://bariya001.github.io/personal/</a></p>
<p>thanks a whole lot.</p>
<p>i will follow the guide to get enable the case-sensitive behavior on a per-directory basis.</p>
